model/src/main/kotlin/OrtResult.kt

```diff
@@ -107,17 +107,24 @@ data class OrtResult(
     private val packages: Map<Identifier, PackageEntry> by lazy {
         log.info { "Computing excluded packages..." }
 
+        val includedPackages = mutableSetOf<Identifier>()
```

Comment:
Commit message: Please again *briefly* (one sentence) write in the commit message body what the basic idea behind the new algorithm is, i.e. *what* makes it faster.

Reply:
Did you really address this? The commit explaines what it makes faster, but not how.

Reply:
Sorry, I associated above comment with wrong commit by accident.
I tried to address it with my latest update of the commit message.